livingatlas/pipelines/src/main/java/au/org/ala/pipelines/converters/TsvConverter.java

```diff
@@ -43,10 +43,11 @@ public String converter(T source) {
     keyTermFnList.forEach(
         keyTermFn -> {
           String value = keyTermFn.getFn().apply(source).orElse(keyTermFn.getDefaultValue());
-          value = "\"" + value + "\"";
+          // replace any tabs within string values with single space
+          value = "\"" + value.replaceAll("\t", " ").replaceAll("\"", "") + "\"";
```

Comment:
Should this properly double-quote escape quotes, rather than deleting them?

Reply:
I'm intentionally deleting them to hopefully reduce issues for GBIF or other consumers.